codalab/worker/dependency_manager.py

```diff
@@ -27,7 +27,6 @@
 
 class DownloadAbortedException(Exception):
     """
-
     Exception raised by the download if a download is killed before it is complete
```

Comment:
Let's add a doc page with the workflow, similar to https://github.com/codalab/codalab-worksheets/pull/3882/files#r761226703

Reply:
Should we add a doc page for the worker code in general instead of just the dependency manager?

Reply:
Add a section in https://codalab-worksheets.readthedocs.io/en/latest/Worker-Managers, called "setting up a shared cache", which talks about how you can point the work directory to the right place (a network disk).